en/computer_vision/collision_prevention.md

```diff
@@ -26,7 +26,7 @@ Multiple sensors can be used to get information about, and prevent collisions wi
 If multiple sources supply data for the _same_ orientation, the system uses the data that reports the smallest distance to an object.
 :::
 
-The vehicle restricts the maximum velocity in order to slow down as it gets closer to obstacles, and will stop movement when it reaches the minimum allowed separation.
+The vehicle restricts the current velocity in order to slow down as it gets closer to obstacles and adapts the acceleration setpoint in order to disallow collision trajectories.
```

Comment:
The first bit makes sense to me - you're approaching a wall, so the vehicle reduces the velocity such that you stop before you hit it. 

FMI, what does "adapts the accel. setpoint to disallow collision trajectories" mean?

Assume I'm heading towards a wall at a 45 degree angle, does this mean? I.e. I'm guessing it might mean that the setpoint is adjusted such that you curve into the wall and then move alongside it. Some images showing the behaviour might be nice.

Reply:
Also youtube the video to the PX4 account and include that in the docs too

Reply:
yea, some figures might actually help, but for that i first need feedback on my PR from Mathias, as its not yet set in stone how the compensation works.
Regarding "adapts the accel. setpoint to disallow collision trajectories", see comment below.

Reply:
Note, would still be good to get a video.